cluster-provision/manifests/cna/operator.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     kubevirt.io: ""
   name: cluster-network-addons-operator
-  namespace: cluster-network-addons-operator
+  namespace: cluster-network-addons
```

Comment:
can you after create a PR that will update all providers that use this manifests?

Reply:
I want to create another PR for the okd 4.1 you also need one for the openshift 3.11 ?

Reply:
yep it can be great!

Reply:
Done can you take a look :)